src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -1,15 +1,16 @@
 <template>
-	<div class="tw-flex">
-		<div id="mobile-menu" class="tw-rounded md:tw-hidden
-				tw-z-dropdown tw-drop-shadow tw-absolute tw-top-4 md:tw-top-0"
-		>
-			<loan-search-filter />
-		</div>
+	<div class="tw-flex" @keyup.esc="closeLightbox">
 		<div class="tw-flex tw-flex-col tw-mr-4">
 			<div class="md:tw-hidden tw-mb-3">
-				<kv-button variant="secondary">
+				<kv-button variant="secondary" @click="openLightbox">
 					Filter & Sort
 				</kv-button>
+
+				<kv-lightbox :visible="isLightboxVisible" title="" preventClose
+					noPaddingTop variant="lightbox" @lightbox-closed="closeLightbox"
```

Comment:
This doesn't existing on the new version of KvLightbox you're using. Please remove for now. There is some precedent for when we need to make hard-coded changes to the new lightbox styles. Here's an example which is similar to what you were doing I think, please try to keep a light touch ;)
https://github.com/kiva/ui/blob/main/src/components/Checkout/AutoDepositUpsell.vue#L282

Reply:
@mcstover I'm assuming you're referring to `noPaddingTop`. If not, VSCode is auto formatting that to use hyphens for me, so @georgebs maybe check your auto save actions.

Reply:
Correct `noPaddingTop` was a prop on our older version of KvLightbox, but we're using the version from kv-ui-components now which doesn't have that option. We have to use some hacky override styles if we "really" need to do something beyond the standard.